homework-submission/todo-app/src/todos/element.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faCheck, faTrash, faPencilAlt, faTimes } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { fab } from '@fortawesome/free-brands-svg-icons'
+import {inject, observer} from 'mobx-react';
+
+
+
+ 
+library.add(fab, faCheck, faTrash, faPencilAlt, faTimes);
+
+@inject('TodosStore')
+@observer
+class Element extends Component {
+    state = {
+        id: this.props.id,
+        description: this.props.description,
+        deadline: this.props.deadline,
+        done: this.props.done,
+        editMode: false,
+        temporarySave: this.props.description
+     }
+
+     throwLineHandle = () => {
+        (!this.state.done) ? this.setState({done: true}) : this.setState({done: false});
```

Comment:
Shorter lines are a bit descriptive to read sometimes:
```suggestion
        this.setState({ done: !this.state.done })
```